### ISSUE TITLE
Do not fail if typo3/ or index.php symlink already exists

### ISSUE THREAD

**Issue by mbrodala** [TYPO3/CmsComposerInstallers#35]
If one does a `rm -rf vendor && composer install` errors like this will occur:

```
[InvalidArgumentException]
The symlink target ".../web/index.php" already exists.
```

Or:

```
[InvalidArgumentException]
The symlink target ".../web/typo3" already exists.
```

This could be avoided if e.g. [`readlink()`](http://php.net/readlink) was used to verify existing symlinks already point to the expected target.

Currently one has to do `rm -rf vendor web/index.php web/typo3 && composer install` instead.

**Comment by helhum** [comment]
This should be solved with 1.4.0RC1. Can you verify?

**Comment by mbrodala** [comment]
Yes, works perfectly fine now. :-)
